src/ui.rs

```diff
@@ -924,16 +919,26 @@ impl<'a> UiFrame<'a> {
                         let wrap_token = ui.push_text_wrap_pos(WRAP_POS_TOOLTIP_TEXT_PIXELS);
                         ui.text_colored(self.colors.tooltip_text, "SAVE SCREENSHOT\n\
                         \n\
-                        Opens the dialog for saving the current viewport into a PNG file.");
+                        Opens a system dialog for saving the current viewport into a PNG file.");
                         wrap_token.pop(ui);
                     });
                 }
 
-                ui.separator();
+                status.export_obj = ui.button(
+                    imgui::im_str!("Export OBJ"),
```

Comment:
there is a convention, that labels of buttons which open a new dialog end with a triple dot, so
`Export OBJ...`

Reply:
Another task. Another PR.

Reply:
I only want you to add three dots to this string. What is the issue?

Reply:
Nevermind, I'll do it in here, it is a small thing. I'll also do https://trello.com/c/Z7Rp0yLS/325-reorder-menu-buttons